maps/randomvaults/oldarmory.dmm

```diff
@@ -0,0 +1,53 @@
+"a" = (/turf/space,/area)
+"b" = (/obj/structure/sign/securearea,/turf/simulated/wall/r_wall,/area/vault/oldarmory)
+"c" = (/obj/machinery/door/airlock/vault,/turf/simulated/floor/plating,/area/vault/oldarmory)
+"d" = (/turf/simulated/wall/r_wall,/area/vault/oldarmory)
+"e" = (/turf/simulated/floor/plating,/area/vault/oldarmory)
+"f" = (/obj/machinery/light/built,/turf/simulated/wall/r_wall,/area/vault/oldarmory)
```

Comment:
You've got a light under an r-wall here

Also /obj/machinery/light/built is an empty fixture iirc